docs/reference-capabilities/default-capabilities.md

```diff
@@ -0,0 +1,233 @@
+
+Classes are a little more flexible. The capability returned by a `new` constructor default to a `ref` if you are constructing a `class`. This is true regardless of what the default capability for the class is, but you have the option to specify a different capability by explicitly typing the constructor as e.g `new var create` or `new iso create_iso`.
+
+Consider the following class:
+
+```pony
+class val MyClass
+    let foo: String
+
+    new create_ref(foo': String) =>
+        foo = foo'
+
+    new val create_val(foo': String) =>
+        foo = foo'
+```
+
+This class has two constructors. The first one, `create_ref` returns a `MyClass` with `ref` capabilities because `ref` is always default for class constructors. The second one `create_val` explicitly returns a `MyClass` with `val` capabilities.
+
+_Tip_: Instead of using the default `ref` type for constructors, consider using `iso`. Recall that constructors, like `consume` return their values as an ephemeral type. That means that a constructor typed as `new iso` actually returns a `iso^`. And `iso^` has the terrific property of being convertible to any other type! Consider:
+
+```pony
+class MyIsoClass
+    let foo: String
+
+    new iso create(foo': String) =>
+        foo = foo'
+```
+
+This class can be constructed as a ref or val or whatever else:
+
+```pony
+a_ref: MyIsoClass ref = MyIsoClass.create()
+a_val: MyIsoClass val = MyIsoClass.create()
+```
+
+In fact, this is what happens if you do not specify a constructor at all. The hidden constructor for this class also returns an `iso^`:
+
+```pony
+class AllDefaults
+    var foo: String = "hello"
+```
+
+You can construct instances of this class with a specific capability by specifying the type of the result:
+
+```pony
+let a = AllDefaults // a is iso
+let b: AllDefaults ref = AllDefaults // b is ref
+let c: AllDefaults val = AllDefaults // c is val
+```
+
+Constructors create objects, but what about when you access functions on those objects?
+
+## Default receiver capabilities
+
+Recall that when you call a function or behaviour, it always implicitly gets a reference to the `this` object.
+
+The default receiver capability for a function is `box`, as shown in `get_foo` below:
+
+```pony
+class MyClass
+    var foo: String = ""
+
+    fun get_foo() => foo
+
+    fun ref set_foo(foo': String) => foo = foo'
+```
+
+Notably, you can't mutate a `box`, so mutable methods must be defined as `fun ref` or `fun trn`.
+
+The _only_ behavior capability for an actor is `ref`. You cannot specify a `be val` for example, or even `be ref`. This may be counter-intuitive since the _caller_ only has a tag to the actor, but the actor is totally free to reference its own fields from the behaviour.
```

Comment:
Is this true? I have never thought to check this so if it is then I just learned something! I cannot think of a reason for it not to be true besides the implementer wanting to create some form of "pass through" behavior which is statically guaranteed not to mutate internal state, but would be allowed to send messages (perhaps will copies of internal state).

Reply:
No it's not true. I understand the idea trying to be expressed, but it isn't true.

This is a misunderstanding of what is happening with viewpoint adaptation. `be` is not `ref`. If it was, then you couldn't send a message to an actor because all you have is a `tag`.

There's other points of  "kind of right, but not quite right" in the explanations here that I was saving to address later.

Reply:
ah, yes; I never thought about how defaults are affected by viewpoints. Most of this came from "instantiate a thing and see what the compiler thinks it is if I assign it to something else (as opposed to "I know what's going on" ;-) ). My eyes may have glossed over a `->` somewhere.

Reply:
I'm not touching this paragraph. I can see that it' wrong, but I think @SeanTAllen will need to set it right.